docs/modules/idrac_system_erase.rst

```diff
@@ -0,0 +1,188 @@
+.. _idrac_system_erase_module:
+
+
+idrac_system_erase -- Erase system and storage components of the server
```

Comment:
Delete system components and storage components of the server.

Reply:
Done